buildout.cfg

```diff
@@ -1,7 +1,14 @@
 [buildout]
 extends = https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.1.x.cfg
+extensions = mr.developer
 package-name = plone.app.z3cform
 package-extras = [tests]
+auto-checkout = plone.app.textfield
+sources-dir = devel
 
 [versions]
 plone.app.z3cform =
+zc.buildout = 3.0.0b3
+
+[sources]
+plone.app.textfield = git git@github.com:gotcha/plone.app.textfield.git branch=ckeditor_1.2
```

Comment:
I have released `plone.app.textfield` 1.2.12, so you can use that.

Reply:
See new commits